src/classes/Logger.cls

```diff
@@ -0,0 +1,36 @@
+/**
+ * Created by Mariya on 14.12.2022.
+ */
+
+public with sharing class Logger {
+    public List<Log__c> logs = new List<Log__c>();
+
+    public void createErrorLog(String summary, String className, Exception e) {
```

Comment:
MINOR
Better name would be: `logError` or 'logSuccess'
Word `create` is mostly used when there is a DML operation in the method, and the consumer expects that.